docs/spec.md

```diff
@@ -252,3 +252,74 @@ const decoded = JSON.parse(new TextDecoder().decode(decrypted.plaintext));
+* `connectionType` is `{"system": "https://smarthealthit.org", "code": "shl-interactive-experience"}`
+* `address` is the URI for the interactive experience
+* `period` optionally documents the window of time when the interactive experience is available
+  * `end` SHALL be populated if there is a time limit
+
+For example, the manifest for an SHL that offers the user the opportunity to "Review a case"
+might include a `application/fhir+json` entry with:
+
+```json
+{
+  "resourceType": "Endpoint",
+  "status": "active",
+  "name": "Review and comment on Alice's case in ACME Medical Diagnostic Portal",
+  "address": "https://interact.example.org/case-id/521039c3-4bb9-45bd-8271-6001d2f4dea9",
+  "period": {"end": "2022-10-20T12:30:00Z"},
+  "connectionType": {"system": "https://smarthealthit.org", "code": "shl-interactive-experience"},
+  "payloadType": [{"system": "http://terminology.hl7.org/CodeSystem/endpoint-payload-type", "code": "none"}],
+}
+```
+
+Notes:
+
+* There is no perfect FHIR resource for documenting an interactive experience URL. `Endpoint` and
+`DocumentReference` are both plausible candidates, and we recommend `Endpoint` here because
+`DocumentReference` is designed for static payloads.
+* If the *only* content being shared via SHL is a single interactive experience, implementers 
+might consider sharing the interactive experience URL directly, instead of through SHL. However,
+since SHL provides a consistent pattern that users and tools can recognize, starting with SHL provides
+a foundation to support future expansion.
+
+
+##### "Upgrading" from SHL to a consumer-mediated SMART on FHIR Connection
+
+In addition to providing direct access to a pre-configured data set, SHLs can include information
+to help establish a consumer-mediated SMART on FHIR connection to the data source. This can be
+accomplished with a SHL manifest entry of type `application/fhir+json` that provides a
+[FHIR Endpoint resource](https://hl7.org/fhir/endpoint.html) where:
+
+* `name` describes the SMART on FHIR endpoint with sufficient detail for the SHL Recipient to decide whether to connect
+* `connectionType` is `{"system": "http://terminology.hl7.org/CodeSystem/restful-security-service", "code": "SMART-on-FHIR"}`
+* `address` is the FHIR API base URL of the server that supports SMART App Launch
+
+For example, the manifest for an SHL from Labs-R-Us might include a `application/fhir+json` entry with:
+
+```json
+{
+  "resourceType": "Endpoint",
+  "status": "active",
+  "name": "Labs-R-Us",
+  "address": "https://fhir.example.org",
+  "connectionType": {"system": "http://terminology.hl7.org/CodeSystem/restful-security-service", "code": "SMART-on-FHIR"},
+  "payloadType": [{"system": "http://terminology.hl7.org/CodeSystem/endpoint-payload-type", "code": "none"}],
+}
+```
+
+Notes:
+
+* Clients may need to pre-register with the SMART on FHIR service before they can request a connection
```

Comment:
```suggestion
* Clients may need to pre-register with the SMART App Launch enabled service before they can request a connection. A client might compare `"address"` against an internal database to determine whether it can connect, retrieve `{address}/.well-known/smart-configuration` to determine whether the [Dynamic Client Registration Protocol](http://hl7.org/fhir/smart-app-launch/client-confidential-asymmetric.html) is available or come up with another way to determine connectivity in order to inform the user of how they can act on the SHL.
```

Reply:
I'm thinking it could be worthwhile expanding on this point for implementers new to the SMART ~~on FHIR~~App Launch world.